### PR TITLE
[Docs] Update change log for v1.6.2

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

File: docs/feature-store/transformations.md [hunk 1]
```diff
@@ -215,7 +215,7 @@ steps. When implementing custom transformations, the code has to support all eng
 
 ```{admonition} Note
 The vast majority of MLRun's built-in transformations support all engines. The support matrix is available 
-[here](../serving/available-steps.html#data-transformations).
+[here](../serving/available-steps.html##data-transformation-steps).
```
Comment (liranbg) [inline docs/feature-store/transformations.md]: does it need double #?

File: docs/change-log/index.md [hunk 2]
```diff
@@ -921,6 +938,7 @@ with a drill-down to view the steps and their details. [Tech Preview]
 
 | Will be removed|Deprecated|API                                                                                |Use instead                                                                                                                                                 |
 |---------------|------------|----------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| v1.8.0       |v1.6.1    |`FunctionSpec.clone_target_dir`                                                      |`ImageBuilder.source_target_dir`
```
Comment (liranbg) [inline docs/change-log/index.md]: can be removed

File: docs/change-log/index.md [hunk 3]
```diff
@@ -849,7 +865,7 @@ with a drill-down to view the steps and their details. [Tech Preview]
 |ML-3143/ML-3432|Cannot delete a remote function from the DB (neither with SDK nor UI).  |NA |v1.2.1    |
 |ML-3445|`project.deploy_function` operation might get stuck when running v1.3.0 demos on an Iguazio platform running v3.2.x.| Replace code: `serving_fn = mlrun.new_function("serving", image="python:3.9", kind="serving", requirements=["mlrun[complete]", "scikit-learn~=1.2.0"])` with: <br>`function = mlrun.new_function("serving", image="python:3.9", kind="serving") function.with_commands([ "python -m pip install --upgrade pip", "pip install 'mlrun[complete]' scikit-learn==1.1.2", ])`|v1.3.0    |
 |NA|The feature store does not support schema evolution and does not have schema enforcement.| NA| v1.2.1    |
-|ML-3521|Cannot schedule a workflow without a remote source. | NA| v1.2.1    |
```
Comment (liranbg) [inline docs/change-log/index.md]: why?

File: docs/change-log/index.md [hunk 4]
```diff
@@ -10,7 +10,23 @@
...
+- [Deprecations and removed code](#deprecations-and-removed-code)
+ 
+
+
+## v1.6.2 (29 March 2024)
+
+###  Closed issue
+| ID          |Description                                                               |
+|----------|---------------------------------------------------------------------------|
+|ML-5808|Changing the project owner to a user that shares a name with another user is successful.|
```
Comment (liranbg) [inline docs/change-log/index.md]: Fix selecting project owner user

File: docs/change-log/index.md [hunk 5]
```diff
@@ -10,7 +10,23 @@
...
+|ML-5982|A user with the 'project security admin' policy can now add her/himself to a project.|
```
Comment (liranbg) [inline docs/change-log/index.md]: can be omitted as it is fixed by ML-5907

File: docs/change-log/index.md [hunk 6]
```diff
@@ -10,7 +10,23 @@
...
+|ML-5907|"Invite New Members" now returns the full list of users when there are 100+ users in system.|
+|ML-5749, 6037|After the user removes ownership of the currently displayed project, the UI redirects to the Projects page.|
+|ML-5977|The 'Members' tab in Project settings is now shown for groups with admin privileges.|
+|ML-5982|A user with 'project security admin' policy can now add her/himself to a project.|
```
Comment (liranbg) [inline docs/change-log/index.md]: dup, remove

File: docs/change-log/index.md [hunk 7]
```diff
@@ -10,7 +10,23 @@
...
+|ML-6025|A group with 'Project security admin' policy can now see the Members tab of Projects.|
+|ML-6036|Changing the project owner |
```
Comment (liranbg) [inline docs/change-log/index.md]: remove

**Review by liranbg — CHANGES_REQUESTED** [review]

File: docs/change-log/index.md (hunk 5 above)
Comment (jillnogold) [reply docs/change-log/index.md]: @liranbg remove both 5808 and 5982?

File: docs/change-log/index.md (hunk 7 above)
Comment (jillnogold) [reply docs/change-log/index.md]: @liranbg remove both 6025 and 6036?

File: docs/change-log/index.md (hunk 5 above)
Comment (liranbg) [reply docs/change-log/index.md]: remove only 5982

File: docs/change-log/index.md (hunk 7 above)
Comment (liranbg) [reply docs/change-log/index.md]: yes

**Review by liranbg — APPROVED** [review]

File: docs/change-log/index.md [hunk 8]
```diff
@@ -884,7 +897,9 @@ with a drill-down to view the steps and their details. [Tech Preview]
 |ML-5079|Cannot update git remote with `project.create_remote()`| NA | v1.5.1 |
 |ML-5204|The **Projects>Settings** does not validate label names. Errors are generated from the back end. |Use [Kubernetes limitations](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set).  | v1.6.0 |
 |ML-5732|When using an MLRun client previous to v1.6.0, the workflow step status might show completed when it is actually aborted.|Abort the job from the SDK instead of from the UI, or upgrade the client. |1.6.0|
-|ML-5776|Concurrent request to project deletion may fail thought first call would gracefully finish the flow, without experiencing any error. Other concurrent requests would not impact the project deletion flow.|NA| v1.6.0|
+|ML-6048|UI: An admin user cannot change its role in the project. |   |v1.6.2|
```
Comment (liranbg) [inline docs/change-log/index.md]: wa?

File: docs/change-log/index.md [hunk 9]
```diff
@@ -884,7 +897,9 @@ with a drill-down to view the steps and their details. [Tech Preview]
...
+|ML-6045|UI: If a user-filter has a large number of matches, it may not display all the matching users. |Narrow your search to be sure you get all the matches.|v1.6.2|
+|ML-6051|UI: After an admin user deletes itself from a project, the user stays in the Projects Members page even though it has no permissions and cannot view any aspects of the project.||  NA |v1.6.2|
```
Comment (liranbg) [inline docs/change-log/index.md]:
```suggestion
|ML-6051|UI: After an admin user deletes itself from a project, the user stays in the Projects Members page even though it has no permissions and cannot view any aspects of the project.|  NA |v1.6.2|
```

**Review by alonmr — APPROVED** [review]

**Review by liranbg — APPROVED** [review]

**Pull request merged**